src/replica/replica_stub.h

```diff
@@ -270,6 +274,39 @@ class replica_stub : public serverlet<replica_stub>, public ref_counter
 
     fs_manager *get_fs_manager() { return &_fs_manager; }
 
+    template <typename TReqType, typename TRespType>
+    bool check_status_and_authz_with_reply(const TReqType &request,
+                                           ::dsn::rpc_replier<TRespType> &reply,
+                                           const ::dsn::ranger::access_type &ac_type)
+    {
+        if (!_access_controller->is_enable_ranger_acl()) {
```

Comment:
`is_enable_ranger_acl()` could also be declared as const ?

Reply:
It is const.